### ISSUE TITLE
Name conflict between different spellings of same nick

### ISSUE THREAD

**Issue by sagamusix** [demozoo/demozoo#274]
Complicated title, easy example: [Teis](https://demozoo.org/sceners/757/) has an alternative spelling "tEiS". Assigning him to some credits now shows a name conflict between... Teis and tEiS! But that is uselsss, because both are the same name and the latter will automatically be corrected to the former. If two nicks "resolve" to the same name, they should not create a conflict.

**Comment by gasman** [comment]
There's no point adding alternative spellings that only differ in capitalisation... the purpose of alternative spellings is to help in matching the right person/group when a 'creative' spelling of their name is used. Since matching happens case-insensitively, having "tEiS" listed won't help to match anything that isn't matched already.

(If "tEiS" is his _preferred_ capitalisation, then we should set that as the primary nick so that we're using that version consistently, and _not_ have "Teis" as an alternative.)

We should probably update the 'edit nick' form so that it silently drops alternatives that only differ in capitalisation.

**Comment by sagamusix** [comment]
Yeah, I'm aware that it's not of much use. Preventing this situation from happening at all seems like a sane fix.
I recall the discussion about whether to preserve NiCKS WiTH fUNNy cAPiTALiSATiOn with valid arguments from both sides. Since tEiS always uses this particular capitalisation, should we set "tEiS" as the main nick?

**Comment by gasman** [comment]
> should we set "tEiS" as the main nick?

Yes, we should, and now I have. :-)

**Comment by glennlunder** [comment]
Just to weigh in on this - the way I've handled these, is I've put down the scener's preferred way to writing his handle as his main one, and made a note in the trivia field. Examples;

https://demozoo.org/sceners/90/
https://demozoo.org/sceners/5667/

In both these cases the people in question talked to me about it directly, though, so it's not always the case that we get the originator's insight.
